protocol/optyfi-protocol/vaults.md

```diff
@@ -20,9 +20,9 @@ LP Vaults can accept Single-Asset LP Tokens like cDAI or aUSDC as well as Multi-
 
 There will be a separate vault for each underlying asset and risk profile combination. 
 
-Upon launch \(ETA July 2021\), the following vaults will be deployed. 
+Upon launch, the following vaults will be deployed: 
 
-| Underlying Asset | Risk Profile |
+| **[Underlying Asset](tokens-and-pools.md#tokens)** | **[Risk Profile](risk-framework.md#risk-profiles)** |
 | :--- | :--- |
 | DAI | RP1, RP2 |
```

Comment:
Change:
WETH (instead of DAI)

Reply:
Under everything:
Progressively, in the Beta and Full launches, more vaults will be deployed incrementally by the OptyFi protocol.